lib/network/modules/InteractionHandler.js

```diff
@@ -165,6 +165,7 @@ class InteractionHandler {
 
   onContext(event) {
     let pointer = this.getPointer({x:event.clientX, y:event.clientY});
+    this.checkSelectionChanges(pointer, event);
```

Comment:
What you're doing here is setting the selection to the node currently hovered over.
This is not what you want; you just want to know over which node the pointer is hovering.

I propose that you use `Selectionhandler.getNodeAt()` instead.